pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -333,8 +337,8 @@ func newRunnerPodForCR(ce chaosTypes.EngineInfo) (*corev1.Pod, error) {
 				WithName("chaos-runner").
 				WithImage(ce.Instance.Spec.Components.Runner.Image).
 				WithImagePullPolicy(corev1.PullIfNotPresent).
-				WithCommandNew([]string{"/bin/bash"}).
-				WithArgumentsNew([]string{"-c", "ansible-playbook ./executor/test.yml -i /etc/ansible/hosts; exit 0"}).
+				WithCommandNew(ce.Command).
```

Comment:
Instead of passing "0" can you create a separate podobj without command and args for go executor and when it's table enough will remove the ansible based "podobj", This will not require any modification in kubehelper